src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link is-active">Apple</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Samsung</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#"
+              class="navigation__link"
+              data-qa="nav-hover">
+            Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Гаджеты</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Планшеты</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Фото</a>
+          </li>
+          <li class="navigation__items-list__item">
+            <a href="#" class="navigation__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="card card--location" data-qa="card">
```

Comment:
Are you sure that this is a modifier ? 
Take a look here [link](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes-en#setting-blocks-external-geometry-or-positioning)
What do you think about that variant ?
```suggestion
    <main class="container">
      <div class="content container__content">
        <div class="card content__card" data-qa="card">
```
The main tag is a semantic block that can contain more than just cards, so you should think about wrapping the block with cards additionally and at the same time it will meet the requirements of the BEM methodology